cou/cli.py

```diff
@@ -59,11 +65,22 @@ def setup_logging(log_level: str = "INFO") -> None:
         fmt="%(asctime)s [%(levelname)s] %(message)s", datefmt="%Y-%m-%d %H:%M:%S"
     )
     root_logger = logging.getLogger()
-    root_logger.setLevel(log_level)
-    if not root_logger.hasHandlers():
-        console_handler = logging.StreamHandler()
-        console_handler.setFormatter(log_formatter)
-        root_logger.addHandler(console_handler)
+    root_logger.setLevel("DEBUG")
+
+    # handler for the log file. Log level is DEBUG
+    pathlib.Path(COU_DIR_LOG).mkdir(parents=True, exist_ok=True)
+    log_file_handler = logging.handlers.TimedRotatingFileHandler(
```

Comment:
We can create separate file for for each run like in snapcraft with cou-yyyyMMddhhmmss-somerandomsuffix.log and show this while starting the application.